r2t2/docstring_reference_parser.py

```diff
@@ -34,25 +31,25 @@ def get_function_reference_identifier(function_reference: FunctionReference) ->
 
 
 def get_function_reference_from_docstring(
-    docstring: CodeDocumentComment
+    docstring: CodeDocumentComment,
 ) -> FunctionReference:
     references = list(iter_parse_plain_text_references(docstring.text))
-    if docstring.lineno != 'n/a':
+    if docstring.lineno != "n/a":
         purpose = DOCSTRING_SHORT_PURPOSE
     else:
         purpose = NOTEBOOK_SHORT_PURPOSE
     return FunctionReference(
-        source=docstring.filename or '',
+        source=docstring.filename or "",
         line=docstring.lineno or 0,
-        name=docstring.name or '',
+        package=docstring.package or "",
```

Comment:
minor: perhaps we should allow `package` to be `None`? (see https://github.com/ImperialCollegeLondon/R2T2/pull/67#issuecomment-686728831)
To be honest, the only reason why I added those `or ""` was because it wasn't marked as optional in `FunctionReference` and I just wanted to get passed the linter.